cpu/stm32l4/include/cpu_conf.h

```diff
@@ -42,8 +44,11 @@ extern "C" {
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
 #if defined(STM32L432KC)
 #define CPU_IRQ_NUMOF                   (82U)
+#elif defined(STM32L496ZG)
+#define CPU_IRQ_NUMOF                   (91U)
```

Comment:
you add only 8 additional irq vectors to the vector table, but increment the number of available IRQs by 10 - I have the impression something got out of sync here :-)